src/test/kotlin/uk/gov/dluhc/printapi/rest/GenerateTemporaryCertificateIntegrationTest.kt

```diff
@@ -19,6 +25,10 @@ internal class GenerateTemporaryCertificateIntegrationTest : IntegrationTest() {
     fun `should return forbidden given no bearer token`() {
         webTestClient.post()
             .uri(URI_TEMPLATE, ERO_ID)
+            .body(
```

Comment:
Existing integration tests now need the request body passing in, now that I've added it to the request method signature in the controller.